deployments/jsonnet/conprof/conprof.libsonnet

```diff
@@ -189,6 +189,25 @@ local k = import 'ksonnet/ksonnet.beta.4/k.libsonnet';
       configmap.mixin.metadata.withNamespace(conprof.config.namespace) +
       configmap.mixin.metadata.withLabels(conprof.config.commonLabels),
 
+    statefulset+: {
+      spec+: {
+        template+: {
+          spec+: {
+            volumes:
+              std.map(
+                function(v) if v.name == 'config' then v {
+                  secret:: null,
+                  configMap: {
+                    name: conprof.configmap.metadata.name,
+                  },
+                } else v,
+                super.volumes
+              ),
+          },
+        },
+      },
+    },
+
     secret:: null,
```

Comment:
Do we still need this one?

Reply:
I think it's just convenience so that if you marshal the whole thing it automatically gets excluded. No strong opinion though if you prefer to keep it.

Reply:
Sure, no strong opinion.